addon/routes/application.js

```diff
@@ -1,3 +1,9 @@
 import Route from "@ember/routing/route";
 
-export default class ApplicationRoute extends Route {}
+export default class ApplicationRoute extends Route {
+  afterModel({ id } = {}) {
+    if (typeof id === "string") {
```

Comment:
If it isn't in the context of project then it is `undefined`?

Reply:
Yes exactly